tests/integration/test_main.py

```diff
@@ -404,3 +404,16 @@ def test_config_file_not_found(tmpdir, capsys):
     out, err = capsys.readouterr()
     assert out == expected
     assert err == ""
+
+
+def test_format_option_help(capsys):
+    """Test that help displays list of available formatters
+    and mapping keys."""
+    with pytest.raises(SystemExit):
+        cli.main(["--help"])
+
+    out, err = capsys.readouterr()
+    output = " ".join(out.split())
+    assert "(default, pylint, quiet-filename, quiet-nothing)" in output
+    assert "(code, col, path, row, text)" in output
```

Comment:
kinda weak assertions -- it would be better to assert the actual text rather than `in` if the messaging is important

Reply:
Do you suggest checking the whole `flake --help` output?